Makefile

```diff
@@ -0,0 +1,19 @@
+# Creates .env template if nonexistent and shows steps to completes before starting the container
+init:
+	./share/scripts/init-message.py
+# Starts docker container
+run:
+	docker-compose down && \
```

Comment:
LOL. We can chat about this in our 1-1, but makefile is probably not the best choice for your intent. And this is coming from a guy who has a lot of makefile experience ;-)

Reply:
Awesome, sounds good.  :)